src/components/MyKiva/BadgeModalContentJournal.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 	<div>
 		<p>
-			{{ description }}
+			{{ badge.fields.shareFact }}
```

Comment:
Is this a required field?

Reply:
Nope, but we need to discuss further what field to use. I just chose one that we could use for now.